tests/unit/dataframer/test_dataframer_smmart.py

```diff
@@ -88,27 +97,28 @@ def smmart_local_db():
 
 @pytest.fixture()
 def document_reference_dataframe():
-    # TODO - is this the expected output?
+    """Create user-defined dataframe/row with Observations (and user-defined nomenclature) that have focus/association to this DocumentReference instance"""
     return {'identifier': '9ae7e542-767f-4b03-a854-7ceed17152cb',
-     'resourceType': 'DocumentReference',
-     'id': '9ae7e542-767f-4b03-a854-7ceed17152cb',
-     'status': 'current', 'docStatus': 'final',
-     'date': '2024-08-21T10:53:00+00:00',
-     'specimen_identifier': 'specimen_1234_labA',
-     'specimen_id': '60c67a06-ea2d-4d24-9249-418dc77a16a9',
-     'specimen_collection': 'Breast',
-     'specimen_processing': 'Double-Spun',
-     'sequencer': 'Illumina Seq 1000',
-     'index': '100bp Single index', 'type': 'Exome',
-     'project_id': 'labA_projectXYZ',
-     'read_length': '100',
-     'instrument_run_id': '234_ABC_1_8899',
-     'capture_bait_set': 'Human Exom 2X',
-     'end_type': 'Paired-End', 'capture': 'emitter XT',
-     'sequencing_site': 'AdvancedGeneExom',
-     'construction': 'library_construction',
-     'Gene': 'TP53', 'Chromosome': 'chr17',
-     'result': 'gain of function (GOF)'}
+            'resourceType': 'DocumentReference',
+            'id': '9ae7e542-767f-4b03-a854-7ceed17152cb',
+            'status': 'current', 'docStatus': 'final',
+            'date': '2024-08-21T10:53:00+00:00',
+            'md5': '227f0a5379362d42eaa1814cfc0101b8',
+            'source_path': 'file:///home/LabA/specimen_1234_labA.fq.gz',
+            'contentType': 'text / fastq',
```

Comment:
`text/fastq` no spaces?

Reply:
also, add title here too? I see it above in `simplified_smmart_resources`.

But more generally, I don't think this should be a fixture on it's own, I think the `simplified_smmart_resources` should be referenced by the DocumentReference key provided